### DISCUSSION TITLE
Can pnpx be used with linked packages?

### DISCUSSION THREAD

**Discussion by marr** [pnpm/pnpm#7130] [Q&A]
I have a package linked in my project, and I was hoping to run a command from it. If I try to do:
`pnpx my-linked-package server` for example, it seems that pnpx doesn't resolve the `my-linked-package` to my locally linked version.
Instead it appears to use the version from the net.

**Comment by stevenpetryk** [comment]
If you're doing this from a package.json script or something, just replace `pnpx` with `pnpm run` to resolve from the cwd.
If you're doing this from your shell, then the answer is no, that would be silly.
